README.md

```diff
@@ -34,6 +34,28 @@ pnpm dev
 # http://localhost:3333
 ```
 
+## docker
+
+You can either choose to pull the image from docker hub or build your own one locally.
+
+### pull image from docker hub
+
+```bash
+# pull the image from docker hub
+docker pull yourname/cook:dev
+# start the container, then open http://localhost:3333
+docker run -it -d --name cook_dev -p 3333:3333 henryclw/cook:dev
```

Comment:
Can you use `yunyoujun` instead of `henryclw` ?

Reply:
Resolved in c6be3b7a